### ISSUE TITLE
Member access for nested structure is not properly annotated

### ISSUE THREAD

**Issue by jeffli678** [Vector35/binaryninja-api#2123]
For the given source code,
```C
#include <stdio.h>
#include <inttypes.h>
#include <stdbool.h>

struct B
{
    char b1;
    uint64_t b2;
    bool b3;
};

struct A
{
    uint32_t a;
    struct B a2;
    char* a3;
};

int main()
{
    struct B b = {'a', 0x12345678, false};
    struct A a = {0xaaaaaaaa, b, "test"};
    return 0;
}
```
I defined the two structures A and B:
![Screenshot from 2020-11-11 15-52-46](https://user-images.githubusercontent.com/3380590/98784402-04a9c180-2436-11eb-816c-bd330dbe1b28.png)

As we can see from the following screenshot, access to Struct B b is properly annotated. However, access to struct A is not. There are displayed as `var_38 + 0xXX`. I expect them to be displayed as, for example, `var38.a2.b1`

![Screenshot from 2020-11-11 15-48-03](https://user-images.githubusercontent.com/3380590/98784429-10958380-2436-11eb-9b4a-1eb83566b82f.png)

**Comment by jeffli678** [comment]
Update: it also malfunctions in MLIL, which I previously did not know:

![Screenshot from 2020-12-02 14-12-11](https://user-images.githubusercontent.com/3380590/100835468-a4dc8e80-34a8-11eb-9e5f-46e534c2d1ba.png)

**Comment by negasora** [comment]
This seems fixed as of 2.2.2598-dev

![image](https://user-images.githubusercontent.com/3020965/104971686-84678580-59bd-11eb-9f79-5281ccd4ef20.png)
![image](https://user-images.githubusercontent.com/3020965/104971707-8df0ed80-59bd-11eb-9f15-aeae5b0b7f5f.png)

**Comment by jeffli678** [comment]
The real problem of this is: #2234
